packages/mediacenter/kodi-binary-addons/visualization.projectm/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="visualization.projectm"
-PKG_VERSION="2.3.2-Leia"
-PKG_SHA256="e30163b4eb12518deb1b56c488b0a81f7eeb3467d291dad6e28234843fc9dc40"
+PKG_VERSION="2.3.4-Leia"
```

Comment:
there is a matrix branch (+ release)

Reply:
Kodi is still using Leia: https://github.com/xbmc/repo-binary-addons/blob/Matrix/visualization.projectm/visualization.projectm.txt